packages/api/cms-api/src/access-log/access-log.module.ts

```diff
@@ -23,14 +24,18 @@ interface AccessLogModuleOptions {
     ],
 })
 export class AccessLogModule {
-    static forRoot({ shouldLogRequest }: AccessLogModuleOptions): DynamicModule {
+    static forRoot({ shouldLogRequest, enableLoggingInDevelopment = false }: AccessLogModuleOptions): DynamicModule {
```

Comment:
I decided against a more generic disableLogging or enableLogging option to prevent mistakes